install_config/persistent_storage/topics/glusterfs_using_storage.adoc

```diff
@@ -68,21 +68,21 @@ Hello World from GlusterFS!!!
 . Delete the pod, recreate it, and wait for it to come up:
 +
 ----
-# oc delete pod nginx-pod
-pod "nginx-pod" deleted
-# oc create -f nginx-pod.yaml
-pod "nginx-pod" created
+# oc delete pod hello-openshift-pod
+pod "hello-openshift-pod" deleted
+# oc create -f hello-openshift-pod.yaml
+pod "hello-openshift-pod" created
 # oc get pods -o wide
 NAME                               READY     STATUS    RESTARTS   AGE       IP               NODE
-nginx-pod                          1/1       Running   0          9m        10.37.0.0        node1
+hello-openshift-pod                          1/1       Running   0          9m        10.37.0.0        node1
 ----
 
 . Now `curl` the pod again and it should still have the same data as before.
 Note that its IP address may have changed:
 +
 ----
 # curl http://10.37.0.0
-Hello World from GlusterFS!!!
+Hello OpenShift!!!
```

Comment:
The string coming from https://github.com/openshift/openshift-docs/pull/10618/files#diff-492418d8b4b3aa3ae12668611e127d51R55
They should be sync. Update both of them or none of them.

Reply:
@mburke5678, is this the right string?